src/main/kotlin/no/nav/brukerdialog/ytelse/ungdomsytelse/api/domene/OppgittInntektForPeriode.kt

```diff
@@ -0,0 +1,27 @@
+package no.nav.brukerdialog.ytelse.ungdomsytelse.api.domene
+
+import no.nav.k9.søknad.felles.type.Periode
+import java.math.BigDecimal
+import java.time.LocalDate
+import no.nav.k9.søknad.ytelse.ung.v1.OppgittInntektForPeriode as UngOppgittInntektForPeriode
+
+data class OppgittInntektForPeriode(
+    val arbeidstakerOgFrilansInntekt: Int? = null,
+    val næringsinntekt: Int? = null,
+    val inntektFraYtelse: Int? = null,
+    val periodeForInntekt: UngPeriode,
+) {
+    fun somUngOppgittInntektForPeriode(): UngOppgittInntektForPeriode = UngOppgittInntektForPeriode(
+        arbeidstakerOgFrilansInntekt?.let { BigDecimal.valueOf(it.toDouble()) },
```

Comment:
jeg syns det er bedre å bruke  BigDecimal.valueOf(it.toLong()) da det er en int og ikke double som er kilden

Reply:
Enig. 
Opprinnelig var den Double, men endret til Int da frontend sender inn hele tall.